sql/core/src/main/scala/org/apache/spark/sql/execution/command/resources.scala

```diff
@@ -46,3 +51,56 @@ case class AddFile(path: String) extends RunnableCommand {
     Seq.empty[Row]
   }
 }
+
+/**
+ * Return a list of file paths that are added to resources.
+ * If file paths are provided, return the ones that are added to resources.
+ */
+case class ListFilesCommand(files: Seq[String] = Seq.empty[String]) extends RunnableCommand {
+  override val output: Seq[Attribute] = {
+    val schema = StructType(
+      StructField("Results", StringType, nullable = false) :: Nil)
+    schema.toAttributes
```

Comment:
how about `AttributeReference("result", StringType, nullable = false)() :: Nil`? And it will be good if we can check what's the output name hive produces.